python_on_whales/exceptions.py

```diff
@@ -9,20 +9,23 @@ def __init__(
         stdout: Optional[bytes] = None,
         stderr: Optional[bytes] = None,
     ):
-        command_launched_str = " ".join(command_launched)
+        self.docker_command = " ".join(command_launched)
+        self.return_code = return_code
+        self.stderr = stderr.decode() if stderr else ""
+        self.stdout = stdout.decode() if stdout else ""
```

Comment:
A few changes would be needed to improve the user experience :) 

1) All attributes should be typed, it will help users when coding in their IDEs or when using Mypy. Just follow the syntax above
2) We should use more explicit types, things that the user can interact with easily and programmatically. For exemple:
`self.docker_command: List[str] = command_launched` because if you join the list, you will loose valuable information, for exemple, where the arguments start or end if there is a space in them.
3) self.stderr and self.stdout should be None and not "". It will allow the user to differentiate between an stderr not captured, and an empty stderr
```python
if stderr is None:
    self.stderr: Optional[str] = None
else:
    self.stderr: Optional[str] = stderr.decode()
```

Reply:
Thanks for the suggestion.  Done in last version or PR (c810682).